custom/icds_reports/utils/__init__.py

```diff
@@ -471,23 +471,27 @@ def chosen_filters_to_labels(config, default_interval=''):
 
 def zip_folder(pdf_files):
     zip_hash = uuid.uuid4().hex
-    client = get_redis_client()
-    in_memory = cStringIO()
+    icds_file = IcdsFile(blob_id=zip_hash, data_type='issnip_monthly')
+    in_memory = BytesIO()
     zip_file = zipfile.ZipFile(in_memory, 'w', zipfile.ZIP_DEFLATED)
     for pdf_file in pdf_files:
-        file = client.get(pdf_file['uuid'])
-        zip_file.writestr('ICDS_CAS_monthly_register_{}.pdf'.format(pdf_file['location_name']), file)
+        file = IcdsFile.objects.get(blob_id=pdf_file['uuid'], data_type='issnip_monthly').get_file_from_blobdb()
```

Comment:
You can get these in bulk from `IcdsFile.objects.get(blob_id__in=(f['uuid'] for f in pdf_file), data_type='issnip_monthly')`

Reply:
@emord updated